doc/installation.rst

```diff
@@ -32,9 +32,9 @@ Just visit the `Spyder page on Binder`_ to get started using Spyder now.
 
 
 
-======================================
-Installing with Anaconda (recommended)
-======================================
+==============
+Using Anaconda
```

Comment:
Now I think it's necessary to add a note below this header to clarify that this is our recommended method.

Reply:
Added, thanks.

Reply:
Thanks @CAM-Gerlach!